packages/liveblocks-react/src/factory.tsx

```diff
@@ -210,23 +242,50 @@ type RoomContextBundle<
   ];
 
   /**
-   * Returns an object that lets you get information about all the the users currently connected in the room.
+   * Returns an object that lets you get information about all the the users
```

Comment:
Just caught an old typo while reviewing

```suggestion
   * Returns an object that lets you get information about all the users
```